### DISCUSSION TITLE
FR: Add description to matrix blocks

### DISCUSSION THREAD

**Discussion by urbantrout** [craftcms/cms#8754] [Ideas]
It would be nice if we could add blocks to a matrix without any fields but with just a description text. Than something like this would be possible:

![Bildschirmfoto am 2019-11-13 um 11 44 32](https://user-images.githubusercontent.com/10465787/68756740-fb133100-060a-11ea-826c-3037db66a787.png)

Optionally adding a link to the section would improve AX a lot.

**Comment by daltonrooney** [comment]
I agree that this would be nice to have built-in. I currently use the [Entry Instructions plugin](https://plugins.craftcms.com/entry-instructions) to accomplish the same thing. And it supports Markdown so links and styling are no problem.
